packages/cardpay-sdk/sdk/scheduled-payment-module.ts

```diff
@@ -341,10 +341,13 @@ export default class ScheduledPaymentModule {
     gasTokenAddress: string,
     salt: string,
     gasPrice: string,
-    payAt: number | null,
-    recurringDayOfMonth: number | null,
-    recurringUntil: number | null
+    payAt?: number | null,
+    recurringDayOfMonth?: number | null,
+    recurringUntil?: number | null
   ): Promise<number> {
+    if (payAt == null && recurringDayOfMonth == null && recurringUntil == null)
+      throw new Error('When payAt is null, recurringDayOfMonth and recurringUntil must have a value');
+
```

Comment:
Should we guard against the opposite case as well? i.e. a user specifies payAt as well as one or both the recurring options? This input doesn't make sense and should be an error.